libs/design/src/molecules/paginator/paginator.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../scss/daff-util';
+
+:host {
+	display: flex;
+	justify-content: center;
+
+	.daff-paginator {
+		&__element {
+			@include clickable();
+			padding: 5px 10px;
+
+			&--selected {
```

Comment:
Moved up to `&__element/&__page-link`

Reply:
It's already under page-link

Reply:
I mean the `border-radius` style. It should be default on the link, not only when it's selected.

Reply:
done